### PR TITLE
Add Lead Offer - Applications team 

### PR DESCRIPTION
**Pull request by tsolakoua**
Signed-off-by: tsolakou <tsolakou.a@gmail.com>

### CODE REVIEW

File: talent/job-descriptions/offers/lead-applications.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+### [Engineering at source{d}](../engineering-section.md)
+
+### Role
+
+This position is for a Lead Engineer in the Applications team.
+
+- As a Lead Engineer, you will be expected to plan current and future work, provide mentorship, overseeing progress of the team and, in general, be able to make day-to-day work smooth for everyone in the team.
+
+- In terms of coding, you will have a two-fold role: on one hand it's a Data Engineering, building production-level pipelines to collect and extract data from all the git repositories in the world. On the another hand it's applied Machine Learning, by productionizing Machine Learning research and delivering it as part of end-user applications.
```
Comment (mcuadros) [inline talent/job-descriptions/offers/lead-applications.md]: You should add the frontend part, and maybe the ML part it's not that important.

File: talent/job-descriptions/offers/lead-applications.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- You will be expected to have strong backend coding skills in at least one programming language and very good algorithmic skills. Scala coding skills and knowledge about Apache Spark aren't required but will be highly appreciated.
+
+- Building tools for day to day use by software engineers require good understanding of modern development processes and approaches: Unit/Integration testing, scrum, code reviews, CI/CD, Containers, etc.
+
+- As you will be working closely with the Machine Learning team - strong communication skills and interest in Machine Learning, data process, and visualization are more than welcome. 
+
+- You will deal with models built using modern ML stacks written in Python and C++ such as Tensorflow, Tensorboard, CUDA, so familiarity with any of those tools is a plus.
```
Comment (bzz) [inline talent/job-descriptions/offers/lead-applications.md]: I would remove CUDA as well

File: talent/job-descriptions/offers/lead-applications.md (hunk 1 above)
Comment (bzz) [reply talent/job-descriptions/offers/lead-applications.md]: Replacing ML clause with something closer web application would be more accurate and makes perfect sense.

Something a long the lines
`On the another hand it's building web applications to productionize Machine Learning research with on both, backend and frontend sides.`

**Comment by tsolakoua** [comment]
Thanks for the feedback! @bzz should they be reflected as well in the senior role description right?

File: talent/job-descriptions/offers/lead-applications.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+### [Company](../company-section.md)
+
+### [Engineering at source{d}](../engineering-section.md)
+
+### Role
+
+This position is for a Lead Engineer in the Applications team.
+
+- As a Lead Engineer, you will be expected to plan current and future work, provide mentorship, overseeing progress of the team and, in general, be able to make day-to-day work smooth for everyone in the team.
```
Comment (carlosms) [inline talent/job-descriptions/offers/lead-applications.md]: overseeing -> oversee
(expected to... oversee)

Or: _provide mentorship overseeing progress of the team_

File: talent/job-descriptions/offers/lead-applications.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- In terms of coding, you will have a two-fold role: on one hand it's a Data Engineering, building production-level pipelines to collect and extract data from all the git repositories in the world. On the other hand, building web applications to productionize Machine Learning research on both, backend and frontend sides.
+
+- You will be expected to have strong backend coding skills in at least one programming language and very good algorithmic skills. Scala coding skills and knowledge about Apache Spark aren't required but will be highly appreciated.
+
+- Building tools for day to day use by software engineers require good understanding of modern development processes and approaches: Unit/Integration testing, scrum, code reviews, CI/CD, Containers, etc.
```
Comment (carlosms) [inline talent/job-descriptions/offers/lead-applications.md]: require -> requires

**Comment by tsolakoua** [comment]
Thanks @carlosms , your comments are addressed at https://github.com/src-d/guide/pull/243/commits/51b9408d7ffffaa8afdc539e53bcf3932cf02205

**Comment by tsolakoua** [comment]
@bzz Our stack is mentioned in the [section](https://github.com/src-d/guide/blob/51b9408d7ffffaa8afdc539e53bcf3932cf02205/talent/job-descriptions/engineering-section.md) that is contained on top of every role. You can check on [Lever](https://jobs.lever.co/sourced/38461466-b253-4d85-a1bf-caafb85f52e7) how it looks like. Let me know if the section needs to be updated for Applications :)

**Comment by smola** [comment]
ping @mcuadros

**Pull request merged**